### PR TITLE
Activate deactivate tenants support

### PR DESCRIPTION
**Pull request by aliszka**

### CODE REVIEW

File: src/schema/tenantsUpdater.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Connection from '../connection';
+import { CommandBase } from '../validation/commandBase';
+import { Tenant } from '../openapi/types';
+
+export default class TenantsCreator extends CommandBase {
```
Comment (tsmith023) [inline src/schema/tenantsUpdater.ts:5]: Should this be `TenantsUpdater`?

File: package.json [hunk 2]
```diff
@@ -68,6 +69,7 @@
     "husky": "^8.0.3",
     "jest": "^29.4.3",
     "lint-staged": "^13.2.0",
+    "openapi-typescript": "^5.4.1",
```
Comment (antas-marcin) [inline package.json:72]: I would not make this `dev dependency` upgradable, I would just set it to `5.4.1` bc higher versions don't support Swagger 2.0

File: tools/refresh_schema.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -euo pipefail
+
+var="${1:-master}"
```
Comment (antas-marcin) [inline tools/refresh_schema.sh:5]: I would change the `var` name to `branch`

**Review by tsmith023 — APPROVED** [review]
LGTM :)

File: src/schema/tenantsUpdater.ts (hunk 1 above)
Comment (aliszka) [reply src/schema/tenantsUpdater.ts:5]: Right, copy/paste 🤦

File: package.json (hunk 2 above)
Comment (aliszka) [reply package.json:72]: should be fine. `^` allows just for minor and patch updates.

**Review by antas-marcin — APPROVED** [review]

**Pull request merged**